src/database/models/reviews.js

```diff
@@ -55,5 +57,45 @@ module.exports = (sequelize, DataTypes) => {
 		}
 	)
 
+	Review.addHook('afterSave', async (instance, options) => {
+		try {
+			//get resource type
+			const resource = await sequelize.models.Resource.findOne(
```

Comment:
@priyanka-TL why do we have to make one more query, won't the instance object have org id and type?

Reply:
@aks30, the reviews table doesn't include the resource type. We'll need to query the resource table to get the resource type.

Reply:
@priyanka-TL In that case can we try where we are calling review save or update we set resource type as well, it might not save in DB but here in instance object we might get that.

Reply:
@aks30 After updating the reviews table we wont get type if we passed also